.slashrc

```diff
@@ -209,6 +209,10 @@ class MediaPlugin(slash.plugins.PluginInterface):
     from lib.platform import info
     return info()["os"]
 
+  def _get_mem(self):
```

Comment:
Function name is ambiguous.  How about `get_physical_mem_total`, instead?  Maybe good to add comment stating it is  in gigabytes.